Meniga/Meniga-ios-sdk/External/Sync/MNFSynchronization.h

```diff
@@ -62,7 +62,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 +(MNFJob *)synchronizeWithTimeout:(NSNumber *)timeout interval:(NSNumber *)interval completion:(MNFSynchronizationCompletionHandler)completion;
 
-+ (MNFJob*)synchronizeRealmUserWithId:(NSNumber *)realmUserId timeout:(NSNumber*)timeout interval:(NSNumber*)interval completin:(nullable MNFErrorOnlyCompletionHandler)completion;
++ (MNFJob*)synchronizeRealmUserWithId:(NSNumber *)realmUserId
+                         sessionToken:(nullable NSString *)sessionToken
+                              timeout:(NSNumber*)timeout
+                             interval:(NSNumber*)interval
+                            completion:(nullable MNFErrorOnlyCompletionHandler)completion;
+
++ (MNFJob*)synchronizeRealmUserWithId:(NSNumber *)realmUserId
+                              timeout:(NSNumber*)timeout
+                             interval:(NSNumber*)interval
+                            completin:(nullable MNFErrorOnlyCompletionHandler)completion
+MNF_DEPRECATED("Please use +[MNFSynchronization synchronizeRealmUserWithId:sessionToken:timeout:interval:completion] instead.");
```

Comment:
Is there a way to add when it will be deprecated? Like in `1.2.0`?

Reply:
Added